src/components/Todo/AddCard/add-card-action-bar.component.jsx

```diff
@@ -3,7 +3,12 @@ import PropTypes from "prop-types";
 import deleteButton from "assets/images/delete.png";
 import { DELETE_BUTTON } from "common/constants";
 
-const AddCardActionBar = ({ handleAddTask, inputTask, isEmptyError }) => {
+const AddCardActionBar = ({
+  handleAddTask,
+  inputTask,
+  isEmptyError,
+  handleCancelClick,
```

Comment:
callback props should be prefixed with `on`

```suggestion
  inputTask,
  isEmptyError,
  onAddTask,
  onCancelClick,
```